code/game/objects/explosion.dm

```diff
@@ -112,10 +114,11 @@ proc/explosion(turf/epicenter, devastation_range, heavy_impact_range, light_impa
 			var/throw_dir = get_dir(epicenter,T)
 			for(var/obj/item/I in T)
 				spawn(0) //Simultaneously not one at a time

```

Comment:
@Miauw62 it's a visual effect, if you read the comment, it causes each thrown item to be thrown at once (you know, as if they'd all been pushed by a big force, like an explosion)

Without it, they move 1 and a time, which is horrific visually.